flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/ExecutionVertexID.java

```diff
@@ -33,6 +35,10 @@
 
 	private final int subtaskIndex;
 
+	public ExecutionVertexID() {
```

Comment:
Could we introduce a util method `ExecutionGraphTestUtils#createRandomExecutionVertexId()` to avoid pollute production code with test code? And make it true random with a random subtaskIndex as well.